redis/asyncio/cluster.py

```diff
@@ -299,6 +300,7 @@ def __init__(
         ssl_ca_certs: Optional[str] = None,
         ssl_ca_data: Optional[str] = None,
         ssl_cert_reqs: Union[str, VerifyMode] = "required",
+        ssl_verify_flags_config: Optional[List[Tuple[VerifyFlags, bool]]] = None,
```

Comment:
I think it makes sense to support it as two separate parameters, something like `include_verify_flags` and `exclude_verify_flags` with  type `Optional[List[VerifyFlags]]`. URL specification allows to pass parameters as arrays, which will be converted to a list, but tuples is something that beyond the query parameters specification. So it will make it more useful if Redis URL is used.